### PR TITLE
Default 'empty' sting for empty values

### PR DESCRIPTION
**Pull request by naz**
refs #63

- Added 'empty' defaults for properties that might come in as empty values

### CODE REVIEW

File: lib/logging/GhostLogger.js [hunk 1]
```diff
@@ -234,11 +234,11 @@ class GhostLogger {
                     statusCode: err.statusCode,
                     level: err.level,
                     message: err.message,
-                    context: jsonStringifySafe(err.context),
-                    help: err.help,
+                    context: jsonStringifySafe(err.context) || 'empty',
+                    help: err.help || 'empty',
```
Comment (naz) [inline lib/logging/GhostLogger.js]: Was thinking if 'help' should go through `jsonStringifySafe` as well :thinking: it's never used as an object though

Comment (kirrg001) [reply lib/logging/GhostLogger.js]: Hmmmmm yeah maybe?

Comment (naz) [reply lib/logging/GhostLogger.js]: Will add it, even if it's just for consistency sake with the rest

Comment (kirrg001) [reply lib/logging/GhostLogger.js]: 👍

**Review by kirrg001 — APPROVED** [review]

**Pull request merged**